### PR TITLE
lxqt.qps: 1.10.20 -> 2.0.0

### PR DESCRIPTION
**Pull request by romildo**
###### Motivation for this change

Update to version [2.0.0](https://blog.lxqt.org/2019/10/release-qps-2-0-0/)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by romildo** [comment]
@GrahamcOfBorg build lxqt.qps

File: pkgs/desktops/lxqt/qps/default.nix [hunk 1]
```diff
@@ -3,24 +3,25 @@
...
   buildInputs = [ qtbase qtx11extras qttools ];
 
   meta = with lib; {
-    description = "The Qt process manager";
+    description = "Qt based process manager
+";
     homepage = https://github.com/lxqt/qps;
     license = licenses.gpl2;
-    maintainers = with maintainers; [ romildo ];
     platforms = with platforms; unix;
```
Comment (c0bw3b) [inline pkgs/desktops/lxqt/qps/default.nix]:
```suggestion
    platforms = with platforms; linux;
```
Hydra darwin's build has been failing forever

**Review by c0bw3b — COMMENTED** [review]
Builds and runs on nixos

Result of `nix-review pr 71158` [1](https://github.com/Mic92/nix-review)
<details>
  <summary>1 package were build:</summary>

  - lxqt.qps
</details>

File: pkgs/desktops/lxqt/qps/default.nix (hunk 1 above)
Comment (romildo) [reply pkgs/desktops/lxqt/qps/default.nix]: Done.

**Review by c0bw3b — APPROVED** [review]

**Pull request merged**